src/freenet/client/filter/M3UFilter.java

```diff
@@ -87,15 +87,19 @@ public void readFilter(
                 readcount = dis.read(nextbyte);
                 continue;
             }
+            if (isCarriageReturn(nextbyte)) {
```

Comment:
This looks like it changes the behaviour of the filter but I can’t see that the test is being changed in reaction to this, so either there is no test (in which case, please add a test) or the test doesn’t actually test this behvaiour (in which case, please add a test for the behvaviour).